helm-chart/renku-core/templates/deployment.yaml

```
@@ -140,13 +143,16 @@ spec:
           args: ["worker"]
           env:
             - name: REDIS_HOST
-              value: {{ include "call-nested" (list $ "redis" "redis.fullname") }}-master
+              value: {{ template "redis.host" . }}
             - name: REDIS_PORT
               value: "6379"
```

```suggestion
              value: {{ .Values.global.redis.port | default "6379" | quote }}
```

Sorry I just noticed this